tox.ini

```diff
@@ -18,13 +18,13 @@ ignore_outcome =
   firefox: false
 deps =
     gocept.pytestlayer
-    py27: pytest < 5.0
-    py{36,37}: pytest
     pytest-cov
     pytest-flake8
     pytest-rerunfailures
     mock
     gocept.testing
+    py27: pytest < 5.0
+    py{36,37}: pytest
```

Comment:
I see no reason to change this, especially in this PR.

Reply:
This was not intended. It got in through merging with default, I'm gonna correct this.